ATM.py

```diff
@@ -0,0 +1,106 @@
+"""
+
+The script simulates ATM that preforms the following:
+balance check, withdraw cash, deposit cash and
+password change.
+
+"""
+import os
+import sys
+
+
+PATH=1
```

Comment:
Make sure to use spaces between the equal signs. It makes the code more readable (do this everywhere). See section 2.7 https://github.com/osherdp/python-training/blob/master/Style_Guide.md